cfme/tests/platform/test_advanced_config.py

```diff
@@ -0,0 +1,35 @@
+def test_advanced_config_reset_nonleaf(appliance):
+    ''' Check whether we can use "<<reset>>" string to reset non-leaf element
```

Comment:
Required:
please fallow `cfme` guideline about `docstring`
https://cfme-tests.readthedocs.io/en/master/guides/documenting.html#docstrings

Reply:
Sorry. Except of space after ''' I see no problem with my docstring.